dttools/src/Makefile

```diff
@@ -120,7 +121,7 @@ HEADERS_PUBLIC = \
 	rmsummary.h \
 	timestamp.h
 
-LIBRARIES = libdttools.a
+LIBRARIES = libdttools.a lib_io_helper.so
```

Comment:
The _ before lib is not customary. I suggest libio_helper.so. (In particular for linking, the name of the library would be _io_helper.)